packages/test/test-service-load/src/leaderElection.ts

```diff
@@ -38,13 +40,14 @@ export class LeaderElection {
     private runLeaderElection() {
         if (this.leaderId !== undefined && this.leaderId === this.dataStoreRuntime.clientId) {
             this.dataStoreRuntime.submitSignal("leaderMessage", "leaderMessage");
-            this.lastPinged = Date.now();
+            this.updateLastPinged();
         }else if(this.leaderId === undefined) {
             this.logger.sendErrorEvent({eventName: "LeaderUndefinedEventError"});
```

Comment:
let me this to sendTelemetryEvent as well, to keep the test results clean as we have a chance to tune it